frontend/public/components/factory/table.tsx

```diff
@@ -121,6 +121,7 @@ const stateToProps = (
     data = [],
     defaultSortField = 'metadata.name',
     defaultSortFunc = undefined,
+    defaultSortOrder = SortByDirection.asc,
```

Comment:
@spadgett I added this in order to properly be able to sort time-based columns. Please let me know if you have any concerns.